src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -55,12 +55,16 @@ Each issue preference option has its own advantages. You can select any preferen
 <table>
   <thead>
     <tr>
-      <th width={200}>
-        **Issue preference**
+      <th>
+        Issue preference
```

Comment:
Are there supposed to be two screenshots in a row? Is it possible to break them up with a little block of text or some context?

Reply:
Fixed!